### ISSUE TITLE
Precisa verificar o tipo de produto antes de validar

### ISSUE THREAD

**Issue by alexlana** [HasteDesign/Registrations-for-WooCommerce#44]
Olá!

A função validate_registration do class-wc-registrations-cart.php não verifica qual o tipo de produto e retorna erros sobre datas em produtos de outros tipos. Acho que quando a gente cadastra como inscrição e depois muda (pode acontecer...), gera esses erros.

**Comment by allysonsouza** [comment]
Olá @alexlana obrigado pelo apontamento, vamos tentar corrigir assim que possível. Caso tenha conhecimentos e quiser fazer um PR com essa correção, será muito bem vindo também! Obrigado.

**Comment by alexlana** [comment]
Estava usando a versão que está no repositório do WP. Essa do GitHub já evoluiu, né? Estou usando ela agora e não encontrei mais o erro.

**Comment by allysonsouza** [comment]
Sim, evoluiu um pouco, acabei não fazendo o release por falta de tempo, e por querer corrigir mais coisas.
